package.json

```diff
@@ -89,11 +98,11 @@
   "lint-staged": {
     "linters": {
       "*.js": [
-        "npm run lint:js -- --fix",
+        "yarn lint:js --fix",
         "git add"
       ],
       "*.json": [
-        "npm run lint:js -- --fix",
+        "yarn lint:js --fix",
         "git add"
       ],
       "*.md": [
```

Comment:
Unfortunately when I try to use yarn for `*.md` tasks, `yarn precommit` hangs

Reply:
#1610